### PR TITLE
Update troubleshooting-nfs-storage-issues.md

### PR DESCRIPTION
**Pull request by mattsn0w**
Changes proposed in this pull request:

- Added specific configuration for Ubuntu 19.x/20.x NFS servers

- Additional NFS troubleshooting steps and checks.

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @mattsn0w 

This pull request needs review by: (please @mention the reviewer if relevant)

### CODE REVIEW

**Comment by sandrobonazzola** [comment]
Thanks for the contribution! @nirs can you please review?

File: source/develop/troubleshooting-nfs-storage-issues.md [hunk 1]
```diff
@@ -166,3 +188,30 @@ In this case it is a good idea to set the tcp window parameters on the NFS serve
...
+        # while :; do  mount| grep nfs | grep -v rpc_pipefs; done
+* Files are created with correct uid:gid (36:36)
+* Check that sanlock user can  read and write files to the filesystem when mounted manually. 
+* Disable SELinux or AppArmor on the NFS server (as a last resort) doesn't fix the issue.
+
+### Check the logs 
+There are 3 logs to check for storage initialization issues found in /var/log/.
+* vdsm/vdsm.log
+* vdsm/supervdsm.log
+* sanlock.log
```
Comment (nirs) [inline source/develop/troubleshooting-nfs-storage-issues.md:207]: This is very important!

We can add more info about the logs:
- /var/log/vdsm/supervdsm.log - show mount errors
- /var/log/vdsm/vdsm.log - show error when vdsm try modify the mounted export as user vdsm
- /var/log/sanlock.log - show errors when sanlock try to modify the mounted export as user sanlock

File: source/develop/troubleshooting-nfs-storage-issues.md [hunk 2]
```diff
@@ -139,11 +161,11 @@ A new **nfs-check** script is now available to test whether an NFS export is rea
       # systemctl start nfs-server
 
 
-## Workarounds for known issues
+# Workarounds for known issues
```
Comment (nirs) [inline source/develop/troubleshooting-nfs-storage-issues.md:164]: There should be only one heading level 1:

    # Heading level 1

This should be heading level 2, change is not needed.

File: source/develop/troubleshooting-nfs-storage-issues.md [hunk 3]
```diff
@@ -139,11 +161,11 @@ A new **nfs-check** script is now available to test whether an NFS export is rea
...
 
 #### TODO: update for the newer supported releases
 
-### NFS Lockups
+## NFS Lockups
```
Comment (nirs) [inline source/develop/troubleshooting-nfs-storage-issues.md:168]: Same, change not needed

**Review by nirs — CHANGES_REQUESTED** [review]
Thanks!

**Comment by sandrobonazzola** [comment]
This PR also needs to be rebased

**Comment by sandrobonazzola** [comment]
closing due to inactivity, please reopen if you start working again on this PR

**Comment by nirs** [comment]
There is important info here that we need to add.

**Comment by sandrobonazzola** [comment]
@nirs are you handling this?

**Comment by ahadas** [comment]
@nirs I'm closing this one because there's no progress for a while but please feel free to post an update/updated PR if you think it is worth it

**Pull request closed without merging**